libcnb-test/src/container_context.rs

```diff
@@ -1,10 +1,128 @@
-use crate::container_port_mapping;
 use crate::IntegrationTestContext;
-use bollard::container::{LogOutput, RemoveContainerOptions};
+use crate::{container_port_mapping, util};
+use bollard::container::{
+    Config, CreateContainerOptions, LogOutput, RemoveContainerOptions, StartContainerOptions,
+};
 use bollard::exec::{CreateExecOptions, StartExecResults};
+use std::collections::HashMap;
 use std::net::SocketAddr;
 use tokio_stream::StreamExt;
 
+pub struct PrepareContainerContext<'a> {
+    integration_test_context: &'a IntegrationTestContext<'a>,
+    exposed_ports: Vec<u16>,
+    env: HashMap<String, String>,
+}
+
+impl<'a> PrepareContainerContext<'a> {
+    pub(crate) fn new(integration_test_context: &'a IntegrationTestContext) -> Self {
+        Self {
+            integration_test_context,
+            exposed_ports: Vec::new(),
+            env: HashMap::new(),
+        }
+    }
+
+    /// Exposes a given port of the container to the host machine.
+    ///
+    /// The given `exposed_port` is mapped to random ports on the host machine. Use
```

Comment:
There is no such variable called `exposed_port` in the signature of the method. Was this meant to be `port` or else for it to use prose ("exposed port") rather than code backticks?

Reply:
Good catch, this is a leftover from a previous version of the docs. I'll fix it. :)